install/helm/open-match/templates/om-configmap-override.yaml

```diff
@@ -32,6 +32,6 @@ data:
         httpport: "{{ .Values.evaluator.httpPort }}"
     synchronizer:
       enabled: true
-      registrationIntervalMs: 3000ms
-      proposalCollectionIntervalMs: 2000ms
+      registrationIntervalMs: 7500ms
```

Comment:
registrationIntervalMs only covers for how long it takes for all the calls to get into OM. If the director does this in a tight loop - then may be a much shorter registration interval (1s?) might work too?

Reply:
Based on the test with 4k profiles I will set it to 3s.